tools/batchTestGenerator/README.md

```diff
@@ -27,17 +27,28 @@ Endpoint for EKS AMP workspace.
 Cluster name for EKS tests.
 #### --eksregion
 Region for EKS tests.
+#### --include
+List of services to include. If not provided will default to all.
+Valid values are as follows "EKS", "ECS", "EC2", "EKS_ARM64", "EKS_ADOT_OPERATOR", "EKS_FARGATE"
```

Comment:
Can we also include mock tests

Reply:
While it may not be documented clearly the primary use of this tool is to generate batches for use in the `Batched` CI workflow in the collector. The `local` command merely exists as an easier way to test the `make execute-batch-test` target without any copying needed by the user. I do not believe there is a use case for mock tests within a `test-case-batch` file because it would not integrate with the `make execute-batch-test` script.

Reply:
Also mock tests are not used during the CI workflow.